examples/python/mapping_example.py

```diff
@@ -472,6 +472,23 @@ def distance2_CRy_decomp(
 
 # Also note that higher performance solutions are coded straight into the TKET c++ codebase. This provides advantages, including that Circuit construction and substitution is unncessary (as with python) as the circuit can be directly modified, however the ability to produce prototypes at the python level is very helpful. If you have a great python implementation but are finding some runtime bottlenecks, why not try implementing it straight into TKET (the code is open source at https://github.com/CQCL/tket).
 
+# The `LexiLabellingMethod()` only labels qubits that are involved in multi-qubit interactions. If a qubit has only single qubit gates or no gates at all this method should be combined with a pass for a general placement first.
+
+c = (
+    Circuit(4)
+    .CX(0, 1)
+    .CX(1, 2)
+    .CX(0, 2)
+    .CX(0, 1)
+    .measure_all()
+)
+display.render_circuit_jupyter(c)
+
+# Observe that qubit q[3] in this example is not labelled (it stays q[3]) as it does not interact with other qubits in the circuit, this could be labelled by using a `pytket.routing.Placement` first.
```

Comment:
```suggestion
# Observe that qubit q[3] in this example is not labelled (it stays q[3]) as it does not interact with other qubits in the circuit. This could be labelled by using a `pytket.routing.Placement` first.
```

Reply:
Also, "pytket.routing.Placement doesn't exist anymore (the routing module is no more).

Reply:
Oh, I will change that. I got trapped by the public docs...